api/app/Services/Leaderboard/Leaderboard.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Services\Leaderboard;
+
+use App\User;
+use App\Vote;
+
+class Leaderboard
+{
+    const CORRECT = 10;
+    const WRONG = -10;
+
+    const SKIP = 0;
+    const VOTE = 1;
+
+    public function forAllUsers()
```

Comment:
It can be cached, but is not needed atm. 
Also, we can't just store it because what will you do when we want a leaderboard per month?